src/Tribe/Plugin.php

```diff
@@ -0,0 +1,123 @@
+<?php
+namespace Tribe\Extensions\Event_Link_To_Website_URL;
+
+/**
+ * Class Plugin
+ *
+ * @since   1.1.0
+ *
+ * @package Tribe\Extensions\Event_Link_To_Website_URL
+ */
+class Plugin extends \tad_DI52_ServiceProvider {
+	/**
+	 * Stores the version for the plugin.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @var string
+	 */
+	const VERSION = '1.1.0';
+
+	/**
+	 * Stores the base slug for the plugin.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @var string
+	 */
+	const SLUG = 'link-events-to-website-urls';
```

Comment:
Is this slug ok? I've seen we use the ext prefix in other places.

Reply:
https://github.com/mt-support/tribe-ext-link-events-to-website-urls/pull/1/files#diff-1467280b57401e476e79ea75fecea470R30 uses a different slug too

Reply:
The slugs shouldn't be prefixed, but the text domain does. I think it's a problem on the Template that assumes they are the same variable. So `SLUG` is correct to not have the prefix.